library/src/main/java/com/datatorrent/lib/io/fs/FSInputModule.java

```diff
@@ -220,7 +243,9 @@ public void setBlockSize(long blockSize)
   /**
    * Gets readers count
    * @return readersCount
+   * @Deprecated Use getMinReaders(), getMaxReaders() instead
```

Comment:
I retained older parameter for maintaining backward compatibility. Other PR #439  would mark this class as evolving. Then we can remove the older parameter.